app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  validates :title, :body, presence: true
+  validates :title, uniqueness: true
+
+  belongs_to :user
+  has_many :comment
```

Comment:
`has_many` association means a post **has many** comment**s** (👈 notice the "s" after comments) i.e., one to many relationship. Please update this so that it follows the Rails convention.